projects/lights_control/src/main.c

```diff
@@ -3,19 +3,101 @@
 #include "can.h"
 #include "gpio.h"
 #include "interrupt.h"
+#include "log.h"
 #include "status.h"
 #include "wait.h"
 
+#include "lights_blinker.h"
+#include "lights_board_type.h"
+#include "lights_can.h"
+#include "lights_can_config.h"
+#include "lights_gpio.h"
+#include "lights_gpio_config.h"
+#include "lights_gpio_config_front.h"
+#include "lights_signal_fsm.h"
+#include "lights_strobe.h"
+
+#define LIGHTS_SIGNAL_BLINKER_DURATION 500
+#define LIGHTS_SIGNAL_SYNC_COUNT 5
+#define LIGHTS_SIGNAL_NO_SYNC 0
+#define LIGHTS_STROBE_BLINKER_DURATION 500
```

Comment:
The naming of these defines is confusing. The way they're worded right now seem to imply they're from the `lights_signal` and `lights_strobe` modules.